src/applications/disability-benefits/all-claims/components/confirmationFields/ConfirmationVeteranInfo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { genderLabels } from 'platform/static-data/labels';
+import { formatDate } from '../../utils/dates/formatting';
+
+export const ConfirmationVeteranInfo = () => {
+  const profile = useSelector(state => state.user?.profile) || {};
+  const { dob, gender } = profile || {};
+  const { first, middle, last, suffix } = profile.userFullName || {};
+
+  return (
+    <li>
+      <div className="vads-u-color--gray">Veteran name</div>
```

Comment:
Whys this one not have a `:`

<img width="573" height="396" alt="Image" src="https://github.com/user-attachments/assets/0260bcbc-01cd-4cc5-8dec-ec3bf138b762" />